stacks-core/mutation-testing/check-packages-and-shards/action.yml

```diff
@@ -41,7 +41,7 @@ runs:
       shell: bash
       id: install_cargo_mutants
       run: |
-        cargo install --git https://github.com/ASuciuX/cargo-mutants --branch feat/variable-test-timeout cargo-mutants # v24.2.0 - with test timeout multiplier implementation
+        cargo install --version 24.2.1 cargo-mutants --locked # v24.2.1
```

Comment:
preferably, we should stay consistent here (ref: https://github.com/stacks-network/actions/blob/feat/mutation-testing/stacks-core/cache/cargo/action.yml#L75-L82)
is there a benefit here to running this in a shell?

Reply:
got it, thanks for the explanation! this is fine then